feat(release): global stageChanges option & changelog fixes (#21223)
nrwl/nx: docs/generated/cli/release.md

Preview the changes without updating files/creating releases

### first-release

Type: `boolean`

> Indicates that this is the first release for the selected release group. If the current version cannot be determined as usual, the version on disk will be used as a fallback. This is useful when using git or the registry to determine the current version of packages, since those sources are only available after the first release. Also indicates that changelog generation should not assume a previous git tag exists and that publishing should not check for the existence the package to publish.

Reviewer: ```suggestion
Indicates that this is the first release for the selected release group. If the current version cannot be determined as usual, the version on disk will be used as a fallback. This is useful when using git or the registry to determine the current version of packages, since those sources are only available after the first release. Also indicates that changelog generation should not assume a previous git tag exists and that publishing should not check for the existence of the package before running.
```

Type: `boolean`

> Whether or not to stage the changes made by this command. Assumed to be true if git-commit is true.

Reviewer: I think we should be stronger with this language:

```suggestion
Whether or not to stage the changes made by this command. Always treated as true if git-commit is true.
```
